apps/verticalhome/test/marionette/app_launch_visibility_test.js

```diff
@@ -0,0 +1,75 @@
+/* global __dirname */
+'use strict';
+
+var assert = require('assert');
+var AppInstall =
+  require('../../../../apps/system/test/marionette/lib/app_install');
+var iconAppState = require('./lib/icon_app_state');
+var createAppServer = require('./server/parent');
+
+marionette('Vertical Home - Launch visibility', function() {
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var server;
+  setup(function(done) {
+    var app = __dirname + '/fixtures/template_app';
+    createAppServer(app, client, function(err, _server) {
+      server = _server;
+      done(err);
+    });
+  });
+
+  var subject;
+  var system;
+  var appInstall;
+
+  setup(function() {
+    subject = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    appInstall = new AppInstall(client);
+
+    system.waitForStartup();
+    subject.waitForLaunch();
+
+    client.switchToFrame();
+    appInstall.installPackage(server.packageManifestURL);
```

Comment:
Is there a reason to do this and not just launch any arbitrary app (and thus cut the size of this test down)?